packages/dd-trace/src/config.js

```diff
@@ -621,8 +633,14 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       eventTracking: {
         enabled: ['extended', 'safe'].includes(DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING),
         mode: DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING
+      },
+      apiSecurity: {
+        enabled: DD_EXPERIMENTAL_API_SECURITY_ENABLED,
+        // Coerce value between 0 and 1
+        requestSampling: Math.min(1, Math.max(0, DD_API_SECURITY_REQUEST_SAMPLE_RATE))
```

Comment:
When I parsed the configuration I didn't know how I was going to implement the sampling rate algorithm so I thought it would be a good idea to normalize it. Now it's implemented with math.random() but we could change it in the future so having a well established set of values to be work with seems like a good practice. Besides it's only performed just at initialization. WDYT?

Reply:
LGTM :+1: